### PR TITLE
Fix #281: Add license header check

### LINKED ISSUE #281: CI: Add license header check
### Description
Need to add an automated CI task to check for license headers.

Create GH action workflow that runs the `mvn -N license:check` command.

### PR DESCRIPTION
**Pull request by rohanKanojia**
## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] ~I Added [CHANGELOG](../CHANGELOG.md) entry~
 - [ ] ~I have implemented unit tests to cover my changes~
 - [ ] ~I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly~
 - [ ] ~No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report~
 - [ ] ~I tested my code in Kubernetes~
 - [ ] ~I tested my code in OpenShift~

### CODE REVIEW

File: .github/workflows/license.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Java 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Install JKube
+        run: mvn -f pom.xml -B -DskipTests clean install
```
Comment (manusa) [inline .github/workflows/license.yml:35]: Why?

File: .github/workflows/license.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+name: License
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  verifications:
```
Comment (manusa) [inline .github/workflows/license.yml:24]: rename to license-check

File: .github/workflows/license.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    name: Quickstart Verifications
```
Comment (manusa) [inline .github/workflows/license.yml:25]:
```suggestion
    name: License Check
```

Comment (rohanKanojia) [reply .github/workflows/license.yml:25]: ah, sorry. I was just trying to set up GitHub action quickly and see if it's working or not. I'll make changes accordingly

**Pull request merged**